ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-cvp-ocp-4.9.yaml

```diff
@@ -0,0 +1,39 @@
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.9"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: msp-common-claim
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.9"
+  cron: '@yearly'
+  steps:
+    dependency_overrides:
+      BUNDLE_IMAGE: brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-11
+      OO_INDEX: brew.registry.redhat.io/rh-osbs-stage/iib:40523
+    env:
+      OO_CHANNEL: "4.3"
+      OO_INSTALL_NAMESPACE: '!create'
+      OO_PACKAGE: e2e-test-operator
+      OO_TARGET_NAMESPACES: '!install'
+      PYXIS_URL: ""
+    workflow: optional-operators-cvp-common-claim
+zz_generated_metadata:
+  branch: ocp-4.9
+  org: redhat-openshift-ecosystem
+  repo: msp
```

Comment:
We can remove it . 
Need to follow up with DPTP. 
Remove it and run make jobs inside the repo check it out.